<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmushClient</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="world_tabs">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_copy_as_html"/>
    <addaction name="action_paste"/>
    <addaction name="separator"/>
    <addaction name="action_select_all"/>
    <addaction name="action_save_selection"/>
    <addaction name="separator"/>
    <addaction name="action_find"/>
    <addaction name="action_find_again"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_status_bar"/>
    <addaction name="separator"/>
    <addaction name="action_wrap_output"/>
    <addaction name="action_pause_output"/>
   </widget>
   <widget class="QMenu" name="menu_connection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="action_disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_auto_connect"/>
    <addaction name="action_reconnect_on_disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_connect_to_all_open_worlds"/>
   </widget>
   <widget class="QMenu" name="menu_game">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="action_go_to_line"/>
    <addaction name="separator"/>
    <addaction name="action_command_history"/>
    <addaction name="separator"/>
    <addaction name="action_clear_output"/>
    <addaction name="separator"/>
    <addaction name="action_reset_all_timers"/>
    <addaction name="separator"/>
    <addaction name="action_stop_sound_playing"/>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="action_maximize"/>
    <addaction name="action_minimize"/>
    <addaction name="action_restore"/>
    <addaction name="separator"/>
    <addaction name="action_new_window"/>
    <addaction name="separator"/>
    <addaction name="action_close_window"/>
    <addaction name="separator"/>
    <addaction name="action_close_all_notepad_windows"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_visit_bug_reports"/>
    <addaction name="action_visit_api_guide"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open_world"/>
    <addaction name="menu_open_recent_world"/>
    <addaction name="action_close_world"/>
    <addaction name="separator"/>
    <addaction name="action_save_world_details"/>
    <addaction name="action_save_world_details_as"/>
    <addaction name="action_edit_world_details"/>
    <addaction name="separator"/>
    <addaction name="action_plugins"/>
    <addaction name="separator"/>
    <addaction name="action_edit_script_file"/>
    <addaction name="action_reload_script_file"/>
    <addaction name="separator"/>
    <addaction name="action_log_session"/>
    <addaction name="action_global_preferences"/>
    <addaction name="separator"/>
    <addaction name="action_print"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
    <addaction name="separator"/>
    <addaction name="action_rec_1"/>
    <addaction name="action_rec_2"/>
    <addaction name="action_rec_3"/>
    <addaction name="action_rec_4"/>
    <addaction name="action_rec_5"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_view"/>
   <addaction name="menu_connection"/>
   <addaction name="menu_game"/>
   <addaction name="menu_window"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="MudStatusBar" name="statusBar">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="192">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="192">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>&amp;New World...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open_world">
   <property name="text">
    <string>&amp;Open World...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_close_world">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close World</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_plugins">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pl&amp;ugins...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="action_save_world_details">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save World Details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_world_details_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save World Details &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_save_selection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Selection...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_global_preferences">
   <property name="text">
    <string>&amp;Global Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::PreferencesRole</enum>
   </property>
  </action>
  <action name="action_log_session">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Log Session...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+J</string>
   </property>
  </action>
  <action name="action_edit_world_details">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit &amp;World Details...</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_copy_as_html">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy as HTML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_select_all">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_rec_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">&amp;1 </string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_rec_2">
   <property name="text">
    <string notr="true">&amp;2 </string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_rec_3">
   <property name="text">
    <string notr="true">&amp;3 </string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_rec_4">
   <property name="text">
    <string notr="true">&amp;4 </string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_rec_5">
   <property name="text">
    <string notr="true">&amp;5 </string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_status_bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Status Bar</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+K</string>
   </property>
  </action>
  <action name="action_auto_connect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Auto Connect</string>
   </property>
  </action>
  <action name="action_reconnect_on_disconnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Reconnect On Disconnect</string>
   </property>
  </action>
  <action name="action_connect_to_all_open_worlds">
   <property name="text">
    <string>Connect To All Open &amp;Worlds</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+K</string>
   </property>
  </action>
  <action name="action_command_history">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Command &amp;History...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_pause_output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pause &amp;Output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="action_find">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_find_again">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Find Again</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="action_go_to_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go To Line...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+L</string>
   </property>
  </action>
  <action name="action_clear_output">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear Output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="action_stop_sound_playing">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop Sound Playing</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+B</string>
   </property>
  </action>
  <action name="action_wrap_output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wrap &amp;Output</string>
   </property>
  </action>
  <action name="action_edit_script_file">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Edit Script File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="action_reload_script_file">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Reload Script File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="action_reset_all_timers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset All Timers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="action_new_window">
   <property name="text">
    <string>&amp;New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="action_minimize">
   <property name="text">
    <string>&amp;Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_close_all_notepad_windows">
   <property name="text">
    <string>Close A&amp;ll Notepad Windows</string>
   </property>
  </action>
  <action name="action_maximize">
   <property name="text">
    <string>Maximize</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+Z</string>
   </property>
  </action>
  <action name="action_restore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="action_visit_bug_reports">
   <property name="text">
    <string>Report a &amp;Bug</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y, Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_visit_api_guide">
   <property name="text">
    <string>API Guide</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit SmushClient</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="menu_open_recent_world">
   <property name="text">
    <string>Open Recent World</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="action_close_window">
   <property name="text">
    <string>Close Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About SmushClient</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MudStatusBar</class>
   <extends>QStatusBar</extends>
   <header>mudstatusbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
